esbuild.js

```diff
@@ -0,0 +1,31 @@
+(async () => {
+    const esbuild = require('esbuild');
+    const sassPlugin = require('esbuild-plugin-sass');
+
+    const result = await esbuild.build({
+        logLevel: 'info',
+        entryPoints: ['js/app.js', 'js/charts.js'],
+        bundle: true,
+        outdir: 'web/build',
+        sourcemap: process.argv.includes('--dev'),
+        watch: process.argv.includes('--dev'),
+        minify: !process.argv.includes('--dev'),
+        metafile: process.argv.includes('--analyze'),
+        loader: {
+            '.gif':'file',
+            '.eot':'file',
+            '.ttf':'file',
+            '.svg':'file',
+            '.woff':'file',
+            '.woff2':'file',
+        },
+        target: ['chrome58', 'firefox57', 'safari11', 'edge95'],
+        plugins: [sassPlugin()],
+    })
+    .catch(() => process.exit(1));
```

Comment:
this should rather be handled by a `try/catch` wrapping the whole body of the async function, or by a `.catch` on the result of the async function (which would then also account for exceptions coming from the second `await`)

Reply:
i hope [b51b327](https://github.com/composer/packagist/pull/1281/commits/b51b3271bd33778d7fec2a814f75a1ebd1c1fb71) is what you meant :D

Reply:
it is